palaeontology.csl

```diff
@@ -106,8 +106,12 @@
             <group suffix=".">
               <group>
                 <text variable="title" font-style="italic" text-case="capitalize-first" suffix=". "/>
-                <text value="In" font-style="italic" suffix=" "/>
-                <text macro="editor" suffix=" "/>
+                <choose>
+                  <if variable="editor">
+                    <text value="In" font-style="italic" suffix=" "/>
+                    <text macro="editor" suffix=" "/>
+                  </if>
+                </choose>
                 <group>
```

Comment:
@adam3smith, I edited your comment to put the "`"s on separate lines for proper formatting.

@ms609, let us know if you need help with this.